object-manipulation-3/main.js

```diff
@@ -0,0 +1,167 @@
+. If not, keep the previous player's score as the total score.
+. After cycling through all the scores. Check which score is the highest & which player has that score.
+. Declare the player with the highest score as the winner!
+. If there's a tie, declare a tiebreaker!
+. First, calculate if there was a tie by seeing if multiple player's had the highest score.
+. Then, replace their hands and their scores to reshuffle and deal.
+. Declar the winner!
+*/
+
+var playerList = ['Player 1', 'Player 2', 'Player 3', 'Player 4'];
+var winner = 0;
+var cardDeck = [
+  { rank: 'Ace', suit: 'clubs', points: 11 },
+  { rank: '2', suit: 'clubs', points: 2 },
+  { rank: '3', suit: 'clubs', points: 3 },
+  { rank: '4', suit: 'clubs', points: 4 },
+  { rank: '5', suit: 'clubs', points: 5 },
+  { rank: '6', suit: 'clubs', points: 6 },
+  { rank: '7', suit: 'clubs', points: 7 },
+  { rank: '8', suit: 'clubs', points: 8 },
+  { rank: '9', suit: 'clubs', points: 9 },
+  { rank: '10', suit: 'clubs', points: 10 },
+  { rank: 'Jack', suit: 'clubs', points: 10 },
+  { rank: 'Queen', suit: 'clubs', points: 10 },
+  { rank: 'King', suit: 'clubs', points: 10 },
+  { rank: 'Ace', suit: 'diamonds', points: 11 },
+  { rank: '2', suit: 'diamonds', points: 2 },
+  { rank: '3', suit: 'diamonds', points: 3 },
+  { rank: '4', suit: 'diamonds', points: 4 },
+  { rank: '5', suit: 'diamonds', points: 5 },
+  { rank: '6', suit: 'diamonds', points: 6 },
+  { rank: '7', suit: 'diamonds', points: 7 },
+  { rank: '8', suit: 'diamonds', points: 8 },
+  { rank: '9', suit: 'diamonds', points: 9 },
+  { rank: '10', suit: 'diamonds', points: 10 },
+  { rank: 'Jack', suit: 'diamonds', points: 10 },
+  { rank: 'Queen', suit: 'diamonds', points: 10 },
+  { rank: 'King', suit: 'diamonds', points: 10 },
+  { rank: 'Ace', suit: 'hearts', points: 11 },
+  { rank: '2', suit: 'hearts', points: 2 },
+  { rank: '3', suit: 'hearts', points: 3 },
+  { rank: '4', suit: 'hearts', points: 4 },
+  { rank: '5', suit: 'hearts', points: 5 },
+  { rank: '6', suit: 'hearts', points: 6 },
+  { rank: '7', suit: 'hearts', points: 7 },
+  { rank: '8', suit: 'hearts', points: 8 },
+  { rank: '9', suit: 'hearts', points: 9 },
+  { rank: '10', suit: 'hearts', points: 10 },
+  { rank: 'Jack', suit: 'hearts', points: 10 },
+  { rank: 'Queen', suit: 'hearts', points: 10 },
+  { rank: 'King', suit: 'hearts', points: 10 },
+  { rank: 'Ace', suit: 'spades', points: 11 },
+  { rank: '2', suit: 'spades', points: 2 },
+  { rank: '3', suit: 'spades', points: 3 },
+  { rank: '4', suit: 'spades', points: 4 },
+  { rank: '5', suit: 'spades', points: 5 },
+  { rank: '6', suit: 'spades', points: 6 },
+  { rank: '7', suit: 'spades', points: 7 },
+  { rank: '8', suit: 'spades', points: 8 },
+  { rank: '9', suit: 'spades', points: 9 },
+  { rank: '10', suit: 'spades', points: 10 },
+  { rank: 'Jack', suit: 'spades', points: 10 },
+  { rank: 'Queen', suit: 'spades', points: 10 },
+  { rank: 'King', suit: 'spades', points: 10 }
+];
```

Comment:
Rather than writing these all out by hand, I would make an array of suits and an array of ranks, then loop over those to generate the deck.